biscuit-auth/src/format/mod.rs

```diff
@@ -40,7 +40,7 @@ pub struct SerializedBiscuit {
 }
 
 impl SerializedBiscuit {
-    pub fn from_slice<KP>(slice: &[u8], key_provider: KP) -> Result<Self, error::Format>
+    pub fn from_slice<KP>(slice: &[u8], key_provider: &KP) -> Result<Self, error::Format>
```

Comment:
would this be posible with a `AsRef` to avoid an API breaking change?

Reply:
No problem! 

Sorry for being picky, would you agree to use `Borrow` instead of `AsRef`? Indeed, `Borrow` has more garantee like `In particular Eq, Ord and Hash must be equivalent for borrowed and owned values: x.borrow() == y.borrow() should give the same result as x == y`